_targets/build.project.armv8r52-mps3an536

```diff
@@ -85,6 +85,8 @@ b_build_target() {
 	b_mkscript_preinit
 
 	make -C plo all
+
+	cp "${PREFIX_PROG_STRIPPED}plo-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.elf" "$PREFIX_BOOT/plo.elf"
```

Comment:
I would only change `"$PREFIX_BOOT/plo.elf"` to `"${PREFIX_BOOT}/plo.elf"` to avoid potential incorrect expansion.